### PR TITLE
Detect when a fasttext executable is available in PATH

### PR DESCRIPTION
**Pull request by pabs3**
Also check that fasttext exists in FT_HOME and is executable.

This is useful when using a distro like Debian that
has a package of fasttext available to install.

### CODE REVIEW

**Comment by pabs3** [comment]
Can someone re-run the checks? Seems like they timed out.

**Comment by mpenkov** [comment]
@piskvorky I'm going to merge this as-is because we use the facebook executable in multiple places during the tests. It's easier to use the executable than to store half a dozen models as test data.

**Comment by mpenkov** [comment]
@pabs3 Thank you for your work!

File: gensim/test/test_fasttext.py [hunk 1]
```diff
@@ -44,7 +45,8 @@
 BUCKET = 10000
 
 FT_HOME = os.environ.get("FT_HOME")
-FT_CMD = os.path.join(FT_HOME, "fasttext") if FT_HOME else None
+FT_CMD = shutil.which("fasttext", path=FT_HOME) or \
```
Comment (piskvorky) [inline gensim/test/test_fasttext.py:48]: Single line please.

**Pull request merged**